


export const getAll = async (req, res) => {
    try {

    } catch (err) {
        res.status(500).json({ error: 'Error getting pricing' });
    }
}


export const create = async (req, res) => {
    try {

    } catch (err) {
        res.status(500).json({ error: 'Error creating pricing' });
    }
}